src/utils/vc/helpers.js

```diff
@@ -68,7 +69,10 @@ export function getSuiteFromKeyDoc(keyDoc) {
  * Helper method to ensure credential is valid according to the context
  * @param credential
  */
-export async function expandJSONLD(credential) {
-  const expanded = await jsonld.expand(credential);
+export async function expandJSONLD(credential, options = {}) {
+  const expanded = await jsonld.expand(credential, {
+    ...options,
+    documentLoader: options.documentLoader || defaultDocumentLoader(options.resolver),
```

Comment:
The `options.resolver` is only ever passed from one place, `verifyCredential`. When it is passed from `verifyCredential` it is guaranteed to be ignored because `verifyCredential` also passes a "truthy" `documentLoader`.

Consider defaulting to `defaultDocumentLoader(null)` instead.
Consider not providing a default documentLoader, instead directly pass options to `jsonld.expand`.

Reply:
If this function is to stay as-is, please add an assertion: `assert(!(options.documentLoader & options.resolver));`

Reply:
"Consider not providing a default documentLoader, instead directly pass options to jsonld.expand." doesnt work, see commits in this PR and comment about json-ld lib calling documentLoader as a function if null/undefined

"The options.resolver is only ever passed from one place" perhaps in our SDK currently, but this is a utility method for others to use aswell. it may be used outside of the credential verification, EG: schema validation. the intention with this change was to allow passing either from outside of the core SDK and allow us to potentially do that in future